### DISCUSSION TITLE
Unable to find image 'openroad/flow-scripts:latest' locally

### DISCUSSION THREAD

**Discussion by heaton56** [The-OpenROAD-Project/OpenROAD#2442] [General]
After install, trying to run the docker I get the following. ny ideas?

Unable to find image 'openroad/flow-scripts:latest' locally
docker: Error response from daemon: pull access denied for openroad/flow-scripts, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.

**Comment by maliberty** [comment]
How did you 'install' it?  What command did you run that gave this message?

**Comment by heaton56** [comment]
Thanks for you response, but I decided to do a local install of OpenROAD-flow-scripts instead, since I would have a learning curve with docker.
